src/components/bs5/footer/footer.hbs

```diff
@@ -64,11 +60,7 @@
                                     <script>
                                         window.formioQldCdnVersion = "v2/v2.x.x-latest";
                                     </script>
```

Comment:
Hi @SenChung , could you escape the closing script tag on L62 and L63? 

<\/script>

(...and I'll cancel my PR).

Reply:
I have just removed all the JS out of the Handlebars file.